fractal_tasks_core/cellpose_segmentation.py

```diff
@@ -99,7 +105,7 @@ def segment_FOV(
     # Write some debugging info
     logger.info(
         f"[{well_id}][segment_FOV] START Cellpose |"
-        f" column: {type(column)}, {column.shape} |"
+        # f" x: {type(x)}, {x.shape} |"
```

Comment:
Any special reason to comment this line?

Reply:
My bad. I was working with lists as input to this function temporarily for cellpose input reasons, forgot to uncomment when I switched back to arrays